snow/engine/snowman/bootstrap/bootstrapper.go

```diff
@@ -755,7 +755,7 @@ func (b *Bootstrapper) restartBootstrapping(ctx context.Context) error {
 
 func (b *Bootstrapper) Notify(_ context.Context, msg common.Message) error {
 	if msg != common.StateSyncDone {
-		b.Ctx.Log.Warn("received an unexpected message from the VM",
+		b.Ctx.Log.Info("received an unexpected message from the VM",
```

Comment:
Why was this changed from `Warn` to `Info`? This is still unexpected behavior right?